webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -1192,6 +1192,15 @@ def mocked_get(url, params, **options):
         response = self.client.get(url, {'page': 'meow'})
         eq_(response.status_code, 200)
 
+        # If you cease to be active, access should be revoked automatically
+        user = User.objects.get(id=user.id)
+        user.is_active = False
+        user.save()
```

Comment:
Why not use ``User.objects.filter(id=self.user.id).update(is_active=False)`` here as well?